webpack.config.js

```diff
@@ -4,6 +4,8 @@ var nodeExternals = require('webpack-node-externals');
 var Fs = require('fs')
 var nodeModules = {}
 var ExtractTextPlugin = require('extract-text-webpack-plugin');
+var autoprefixer = require('autoprefixer');
+var precss = require('precss');
```

Comment:
Ah oops good catch. It was another plugin I was playing with to try to get `postcss` to play nicely with Sass, but I found a much better way around it where we could keep `sass-loader`. I'll remove it.